### ISSUE TITLE
Does not work properly with a child view

### ISSUE THREAD

**Issue by markr79** [funkensturm/ember-cli-file-picker#3]
According to the docs, `eventManager` will process events from current instance or any of the descendants. So if you have an ember view inside the file-picker view, troubles are eminent.
That means that 
``` 
  dragEnter: function(event, view) {
      if (!view.get('multiple')) {
        view.clearPreview();
      }

      view.$().addClass('over');
    }
```
will crash in `view.clearPreview()` as current view is not the file picker view but the descendant.

**Comment by fsmanuel** [comment]
@markr79 thanks for info. I guess I will steer clear from `eventManager` and release a new version.

**Comment by markr79** [comment]
Thanks!

**Comment by fsmanuel** [comment]
@markr79 can you test against master?

**Comment by markr79** [comment]
@fsmanuel There are some additional problems with dragEnter/dragLeave when 'over' class is not applied properly. I will create a pull request with a fix.

**Comment by fsmanuel** [comment]
@markr79 i got into that but didn't had the time to fix. thanks for the PR.

**Comment by fsmanuel** [comment]
@markr79 i released `0.0.5` with you fix

**Comment by markr79** [comment]
@fsmanuel great! you may want to update docs for `selectOnClick` property.
